README.md

```diff
@@ -46,13 +46,36 @@ Contributions are welcome - check our [Contribution Guide](https://github.com/ti
 
 ## Installation 
 
-ETNA is on [PyPI](https://pypi.org/project/etna), so you can use `pip` to install it.
+ETNA is available on [PyPI](https://pypi.org/project/etna), so you can use `pip` to install it.
 
+Install default version:
 ```bash
 pip install --upgrade pip
 pip install etna
 ```
 
+Default version hasn't all the dependencies, because some of them are needed only for specific models, e.g. Prophet, PyTorch. 
+Available extensions are listed at [`pyproject.toml`](https://github.com/tinkoff-ai/etna/blob/master/pyproject.toml#L93).
+
+Without the appropriate extension you will get an `ImportError` trying to import the model that needs it.
+For example, `etna.models.ProphetModel` needs `prophet` extension and can't be used without it.
+
+Install extension:
+```bash
+pip install etna[extension-name]
+```
+
+Install all extensions:
+```bash
+pip install etna[all]
+```
+
+### Configuration
+
+It can be useful to configure the library to check installed packages during init.
+For example, if you know that you need `Prophet` in you project and don't want to understand that it isn't installed only during the import of `etna.models.ProphetModel`.
```

Comment:
Maybe this way it will clearer:

etna supports configuration files. It means that etna will check that all the specified packages are installed prior to script start and NOT during runtime.